libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -1071,6 +1071,7 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
         address: "https://explorer.elrond.com/accounts/$address",
       },
     ],
+    keywords: ["elrond"],
```

Comment:
not related to the PR but I think it is necessary. User can type "elrond" to find the currency in LL